source/core/rtc_adapter/RtcAdapter.h

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) <2020> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef RTC_ADAPTER_RTC_ADAPTER_H_
+#define RTC_ADAPTER_RTC_ADAPTER_H_
+
+#include <MediaFramePipeline.h>
+
+namespace rtc_adapter {
+
+class AdapterDataListener {
+public:
+    virtual void onAdapterData(char* data, int len) = 0;
+};
+
+class AdapterFrameListener {
+public:
+    virtual void onAdapterFrame(const owt_base::Frame& frame) = 0;
+};
+
+class AdapterFeedbackListener {
+public:
+    virtual void onFeedback(const owt_base::FeedbackMsg& msg) = 0;
+};
+
+struct AdapterStats {
+    int width;
+    int height;
+    owt_base::FrameFormat format;
+    int estimatedBandwidth;
+};
+
+class AdapterStatsListener {
+public:
+    virtual void onAdapterStats(const AdapterStats& stat) = 0;
+};
+
+class VideoReceiveAdapter {
+public:
+    virtual int onRtpData(char* data, int len) = 0;
+    virtual void requestKeyFrame() = 0;
+};
+
+class VideoSendAdapter {
```

Comment:
RtcAdapter.h should only include the declaration common to video/audio adapter.  The declaration of audio/video adapters should be move to their corresponding headers.